assets/xml/objects/object_mm.xml

```diff
@@ -1,68 +1,80 @@
+        <DList Name="gPostmanHeadDL" Offset="0x5E70" />
+        <DList Name="gPostmanRightShoulderDL" Offset="0x6598" />
+        <DList Name="gPostmanRightArmDL" Offset="0x66D0" />
+        <DList Name="gPostmanRightHandDL" Offset="0x6808" />
+        <DList Name="gPostmanLeftShoulderDL" Offset="0x6A10" />
+        <DList Name="gPostmanLeftArmDL" Offset="0x6B48" />
+        <DList Name="gPostmanLeftHandDL" Offset="0x6C80" />
+        <DList Name="gPostmanWaistDL" Offset="0x6E88" />
+        <DList Name="gPostmanRightThighDL" Offset="0x6F20" />
+        <DList Name="gPostmanRightLegDL" Offset="0x7188" />
+        <DList Name="gPostmanRightFootDL" Offset="0x72F8" />
+        <DList Name="gPostmanLeftThighDL" Offset="0x7430" />
+        <DList Name="gPostmanLeftLegDL" Offset="0x7610" />
+        <DList Name="gPostmanLeftFootDL" Offset="0x7780" />
+        <DList Name="gPostmanLetterDL" Offset="0x8348" /> <!-- Original name is "mm_letter_model" -->
+        <DList Name="gPostmanBagDL" Offset="0x83E0" /> <!-- Original name is "mm_bag_model" -->
+        <DList Name="gPostmanHatDL" Offset="0x85C8" /> <!-- Original name is "mm_cap_model" -->
+        
+        <!-- Postman Limbs -->
+        <Skeleton Name="gPostmanSkel" Type="Flex" LimbType="Standard" LimbNone="POSTMAN_LIMB_NONE" LimbMax="POSTMAN_LIMB_MAX" EnumName="PostmanLimb" Offset="0x96E8" />
+        <Limb Name="gPostmanWaistLimb" Type="Standard" EnumName="POSTMAN_LIMB_WAIST" Offset="0x95F8" />
+        <Limb Name="gPostmanLeftThighLimb" Type="Standard" EnumName="POSTMAN_LIMB_LEFT_THIGH" Offset="0x9604" />
+        <Limb Name="gPostmanLeftLegLimb" Type="Standard" EnumName="POSTMAN_LIMB_LEFT_LEG" Offset="0x9610" />
+        <Limb Name="gPostmanLeftFootLimb" Type="Standard" EnumName="POSTMAN_LIMB_LEFT_FOOT" Offset="0x961C" />
+        <Limb Name="gPostmanRightThighLimb" Type="Standard" EnumName="POSTMAN_LIMB_RIGHT_THIGH" Offset="0x9628" />
+        <Limb Name="gPostmanRightLegLimb" Type="Standard" EnumName="POSTMAN_LIMB_RIGHT_LEG" Offset="0x9634" />
+        <Limb Name="gPostmanRightFootLimb" Type="Standard" EnumName="POSTMAN_LIMB_RIGHT_FOOT" Offset="0x9640" />
+        <Limb Name="gPostmanTorsoLimb" Type="Standard" EnumName="POSTMAN_LIMB_TORSO" Offset="0x964C" />
+        <Limb Name="gPostmanLeftShoulderLimb" Type="Standard" EnumName="POSTMAN_LIMB_LEFT_SHOULDER" Offset="0x9658" />
+        <Limb Name="gPostmanLeftArmLimb" Type="Standard" EnumName="POSTMAN_LIMB_LEFT_ARM" Offset="0x9664" />
+        <Limb Name="gPostmanLeftHandLimb" Type="Standard" EnumName="POSTMAN_LIMB_LEFT_HAND" Offset="0x9670" />
+        <Limb Name="gPostmanRightShoulderLimb" Type="Standard" EnumName="POSTMAN_LIMB_RIGHT_SHOULDER" Offset="0x967C" />
+        <Limb Name="gPostmanRightArmLimb" Type="Standard" EnumName="POSTMAN_LIMB_RIGHT_ARM" Offset="0x9688" />
+        <Limb Name="gPostmanRightHandLimb" Type="Standard" EnumName="POSTMAN_LIMB_RIGHT_HAND" Offset="0x9694" />
```

Comment:
```suggestion
        <Limb Name="gPostmanLeftThighLimb" Type="Standard" EnumName="POSTMAN_LIMB_LEFT_THIGH" Offset="0x9604" />
        <Limb Name="gPostmanLeftShinLimb" Type="Standard" EnumName="POSTMAN_LIMB_LEFT_SHIN" Offset="0x9610" />
        <Limb Name="gPostmanLeftFootLimb" Type="Standard" EnumName="POSTMAN_LIMB_LEFT_FOOT" Offset="0x961C" />
        <Limb Name="gPostmanRightThighLimb" Type="Standard" EnumName="POSTMAN_LIMB_RIGHT_THIGH" Offset="0x9628" />
        <Limb Name="gPostmanRightShinLimb" Type="Standard" EnumName="POSTMAN_LIMB_RIGHT_SHIN" Offset="0x9634" />
        <Limb Name="gPostmanRightFootLimb" Type="Standard" EnumName="POSTMAN_LIMB_RIGHT_FOOT" Offset="0x9640" />
        <Limb Name="gPostmanTorsoLimb" Type="Standard" EnumName="POSTMAN_LIMB_TORSO" Offset="0x964C" />
        <Limb Name="gPostmanLeftUpperArmLimb" Type="Standard" EnumName="POSTMAN_LIMB_LEFT_UPPER_ARM" Offset="0x9658" />
        <Limb Name="gPostmanLeftForarmLimb" Type="Standard" EnumName="POSTMAN_LIMB_LEFT_FORARM" Offset="0x9664" />
        <Limb Name="gPostmanLeftHandLimb" Type="Standard" EnumName="POSTMAN_LIMB_LEFT_HAND" Offset="0x9670" />
        <Limb Name="gPostmanRightUpperArmLimb" Type="Standard" EnumName="POSTMAN_LIMB_RIGHT_UPPER_ARM" Offset="0x967C" />
        <Limb Name="gPostmanRightForarmLimb" Type="Standard" EnumName="POSTMAN_LIMB_RIGHT_FORARM" Offset="0x9688" />
        <Limb Name="gPostmanRightHandLimb" Type="Standard" EnumName="POSTMAN_LIMB_RIGHT_HAND" Offset="0x9694" />
```
Sorry these body parts are fairly standard among human skeletons and these are the names we have mostly settled on.